apps/whale-api/src/module.api/poolpair.fees.controller.e2e.ts

```diff
@@ -0,0 +1,363 @@
+import { PoolPairController } from './poolpair.controller'
```

Comment:
`poolpair.fees.controller.e2e.ts` -> can we call it `poolpair.fees.service.e2e.ts`, since there's no PoolpairFeesController?

Reply:
oh, 🙈 updated!